scenarios/evaluate-safety/README.md

```diff
@@ -0,0 +1,29 @@
+---
+page_type: sample
+languages:
+- language1
+- language2
+products:
+- ai-services
+- azure-openai
+description: Evaluate Protected Material and Indirect Attack Jailbreak
+---
+
+## Evaluate Protected Material and Indirect Attack Jailbreak
+
+### Overview
+
+This notebook walks through how to generate a simulated conversation targeting a deployed AzureOpenAI model and then evaluate that conversation dataset for Protected Material and Indirect Attack Jailbreak vulnerability. It also references the prompt filtering capabilities of Azure AI to help identify and mitigate these vulnerabilities in your AI system.
+
+### Objective
+
+The main objective of this tutorial is to help users understand how to use the promptflow-evals package to simulate a conversation with an AI system and then evaluate that dataset on various safety metrics. By the end of this tutorial, you should be able to:
```

Comment:
Shouldn't this be updated to the new `azure-ai-evaluation` SDK package?

Reply:
Since that package isn't released yet, I figured we didn't want to create a sample for it, but I can change out the references in this notebook since you want to use it in your blog.